internal/lido/contracts/csfeedistributor/rewards_test.go

```diff
@@ -0,0 +1,95 @@
+package csfeedistributor
+
+import (
+	"io"
+	"log"
+	"math/big"
+	"testing"
+
+	bond "github.com/NethermindEth/sedge/internal/lido/contracts/csaccounting"
+)
+
+func TestRewards(t *testing.T) {
+	log.SetOutput(io.Discard)
+	tcs := []struct {
+		name         string
+		network      string
+		nodeID       *big.Int
+		proofStrings []string
+		wantErr      bool
+	}{
+		{
+			name:    "Rewards for nodeID 1, Holesky",
+			network: "holesky",
+			nodeID:  big.NewInt(1),
+			proofStrings: []string{
+				"0x6d20abd8b1d6bd278c57e818ff4984c023509ff6b5ad82d14b04725b19bcc70c",
```

Comment:
I thought the proof can be changed anytime. How come that we can use fixed proof here?

Reply:
I'm not sure how often they change, but wanted to test them to make sure the method works. Until now the values haven't changed.

Reply:
proof parameter has been removed, we found a way to get the rewards without it. Changes adjusted in [this commit](https://github.com/NethermindEth/sedge/pull/401/commits/cb7f2944bcee2c5aad869dab6825b21f8de8162c)